demo/deploy.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+# Create a Certificate Signing Request (CSR) for our admission webhook service
+# See https://kubernetes.io/docs/tasks/tls/managing-tls-in-a-cluster/ for more detail
+CSR_NAME='demo-csr.kube-exec-controller'
+kubectl delete csr $CSR_NAME 2>/dev/null
+rm -rf server*
+cat demo/csr.json | cfssl genkey - | cfssljson -bare server # this generates `server.csr` and `server-key.pem` files
```

Comment:
Can you add the appropriate packages that provide `cfssl` and `cfssljson` to the prereqs?

Reply:
Good catch! I have updated the script to download both tools from [cloudflare/cfssl ](https://github.com/cloudflare/cfssl/releases) (also used in K8s official docs) in #2